Update Dupla_VICTORHUGO_BHRUNNO.md

VictorBSI opened the pull request:
Atualizando com as atividades 5, 6 e 7

VictorBSI committed:
Update Dupla_VICTORHUGO_BHRUNNO.md

Atualizando com as atividades 5, 6 e 7

filipesaraiva reviewed (changes requested):
Colocar os conteúdos das atividades no próprio arquivo de dupla.

VictorBSI committed:
Update Dupla_VICTORHUGO_BHRUNNO.md

Atualizado com as atividades escritas na própria página da dupla e adicionada a atividade 08.

VictorBSI committed:
Update Dupla_VICTORHUGO_BHRUNNO.md

VictorBSI committed:
Update Dupla_VICTORHUGO_BHRUNNO.md

Add Atividade 10
